src/main/java/com/scalar/cassy/config/CassyServerConfig.java

```diff
@@ -95,6 +99,8 @@ private void load() {
     sshPrivateKeyPath = props.getProperty(SSH_PRIVATE_KEY_PATH);
     checkArgument(props.getProperty(SLAVE_COMMAND_PATH) != null);
     slaveCommandPath = props.getProperty(SLAVE_COMMAND_PATH);
+    checkArgument(props.getProperty(STORAGE_TYPE) != null);
+    storageType = StorageType.valueOf(props.getProperty(STORAGE_TYPE));
```

Comment:
To make the config case insensitive.
```suggestion
    storageType = StorageType.valueOf(props.getProperty(STORAGE_TYPE).toUpperCase());
```